analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ReferenceEqualityCheckWhenEqualsExists2.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace AppendedNamespaceForConcurrencyTest
+{
+    namespace Tests.Diagnostics
```

Comment:
Same as on other places, to persist ident 
```suggestion
    namespace AppendedNamespaceForConcurrencyTestTests.Diagnostics
```